weave

```diff
@@ -806,6 +806,19 @@ dns_args() {
     fi
 }
 
+rewrite_hosts() {
+    hosts_path_and_name=$(docker inspect -f '{{.HostsPath}} {{.Config.Hostname}}' $CONTAINER)
+    hosts=$(echo $hosts_path_and_name | cut -f1 -d" ")
+    name=$(echo $hosts_path_and_name | cut -f2 -d" ")
+    containers_path="$(dirname $(dirname "$hosts"))"
+    mnt="/containers"
+    mnt_hosts=$(echo $hosts | sed 's|'$containers_path'|'$mnt'|g')
```

Comment:
related to the `dirname` question... why isn't this just `$mnt/$(basename "$hosts"))`?